From 35ee9e35be06a99b59689bc26f43bf811a399ee6 Mon Sep 17 00:00:00 2001
From: Jin Li <jinl@codeaurora.org>
Date: Thu, 26 May 2016 17:25:25 -0400
Subject: [PATCH 40/44] ARM: dts: support drm/sde in mojave board

Adding HDMI connectors in DTS to support DRM/KMS driver in Mojave board.

Change-Id: Idb81c869fac7c120ad4e75bd6a444dae039456b4
Signed-off-by: Jin Li <jinl@codeaurora.org>
---
 arch/arm/boot/dts/qcom/msm8996-agave-adp.dtsi | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm/boot/dts/qcom/msm8996-agave-adp.dtsi b/arch/arm/boot/dts/qcom/msm8996-agave-adp.dtsi
index 10ad47b..8df9117 100644
--- a/arch/arm/boot/dts/qcom/msm8996-agave-adp.dtsi
+++ b/arch/arm/boot/dts/qcom/msm8996-agave-adp.dtsi
@@ -548,6 +548,18 @@
 
 &mdss_mdp {
 	qcom,mdss-pref-prim-intf = "dsi";
+	connectors = <&mdss_hdmi>;
+	memory-region = <&cont_splash_mem>;
+};
+
+&mdss_hdmi {
+	status = "ok";
+	qcom,hdmi-tx-ddc-clk-gpio = <&tlmm 32 0>;
+	qcom,hdmi-tx-ddc-data-gpio = <&tlmm 33 0>;
+	qcom,hdmi-tx-hpd-gpio = <&tlmm 34 0>;
+	qcom,hdmi-tx-mux-en-gpio = <&tlmm 27 0>;
+	qcom,hdmi-tx-mux-sel-gpio = <&tlmm 83 0>;
+	qcom,hdmi-tx-mux-lpm-gpio = <&tlmm 85 0>;
 };
 
 &mdss_dsi {
-- 
2.8.2

